keps/sig-network/1672-tracking-terminating-endpoints/kep.yaml

```diff
@@ -20,16 +20,18 @@ see-also:
 replaces: []
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: stable
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.22"
+latest-milestone: "v1.23"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.20"
+  beta: "v1.22"
```

Comment:
Did we went Beta in 1.22 without: 
(a) having this tracked appropriately by RT?
(b) having PRR approved?

This sounds very wrong to me...

Reply:
There was a last-minute decision during a SIG Network call to include this feature as beta in v1.22 -- I guess we slipped the KEP updates though. Sorry, that's my bad :(

Reply:
I have no particular recollection of how this came to be, and certainly did not mean to bypass any process.